### ISSUE TITLE
Include basemap attributions on printout

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MassGIS/morisoliver#222]
```
For Digital Globe Orthos basemap we would like the current attribution (which 
does show up in the map window in the bottom right as Copyright symbol Digital 
Globe, Inc. All Rights Reserved) to show up also on the printout. If it doesn't 
work to use the attribution maybe a separate place to hold the string to print. 

```

Original issue reported on code.google.com by `Aleda.Fr...@state.ma.us` on 11 Jul 2013 at 1:55

**Comment by MassGIS** [comment]
Aleda 11/12/2015 Digital Globe is no longer in the basemaps area, because it is no longer the most current aerial photography. It is now in the datalayers folder.  When the map is printed now the label prints from the datalayer area.  

But the question is still the same - we'd like to have on the printout some label of what the basemap is.

**Comment by MassGIS** [comment]
We would like to have titles on the legend for any non-Google basemaps - this currently includes OSM, 2013-2014 orthos and MassGIS Statewide Basemap.  All MassGIS basemaps are currently hosted at arcgis.com

**Comment by cgalvarino** [comment]
Addressed in #260.
